README.md

```diff
@@ -11,44 +11,46 @@ This repo contains two main programs:
 
 ## Installation
 
-pip install git+https://github.com/trompamusic/trompa-ce-client.git@tpl
+    pip install git+https://github.com/trompamusic/trompa-ce-client.git
 
-git clone https://github.com/trompamusic/tpl
+    git clone https://github.com/trompamusic/tpl
 
-cd tpl
-
-pip install -r requirements.txt
-
-python setup.py install
+    cd tpl
 
+    pip install -r requirements.txt
 
 
 ## Processing Library 
 
 To invoke one the TPL for a specific algorithm/software one has to run the following:
 
-python  application.py --ce ce_config_file --app app_config_file --client --clinet_confif_file --force (optional)
+    python -m tpl.application --connection ce_config_file --app app_config_file --client client_config_file [--force] [--execute]
```

Comment:
some of these arguments seem to have changed, so I updated them.

I added the `python -m` syntax to be able to run this during local development as well without having to run `python setup.py install`